dashboard/functions/index.js

```diff
@@ -10,14 +10,19 @@ if (!process.env.GCLOUD_PROJECT) {
   process.env.GCLOUD_PROJECT = PROJECT_ID;
 }
 
+const UDMI_TARGET = process.env.UDMI_TARGET || 'udmi_target';
+const UDMI_STATE = process.env.UDMI_STATE || 'udmi_state';
+const UDMI_REFLECT = process.env.UDMI_REFLECT || 'udmi_reflect';
+const UDMI_CONFIG = process.env.UDMI_CONFIG || 'udmi_config';
+const REFLECT_REGISTRY =  process.env.REFLECT_REGISTRY ||'UDMS-REFLECT';
+
 const version = require('./version');
 const functions = require('firebase-functions');
 const admin = require('firebase-admin');
 const { PubSub } = require(`@google-cloud/pubsub`);
 const pubsub = new PubSub();
 const iot = require('@google-cloud/iot');
 
-const REFLECT_REGISTRY = 'UDMS-REFLECT';
```

Comment:
As per the other issue, should the region for the registry be configurable as well?  It looks like it might be inappropriately assuming the wrong one (that is, the one of the site registry itself), so we might need to fix that... so maybe that is worth a separate PR...